tobac/utils.py

```diff
@@ -894,3 +894,174 @@ def spectral_filtering(
         return (lambda_mn, transfer_function), filtered_field
     else:
         return filtered_field
+
+
+def compress_all(nc_grids, min_dims=2, comp_level=4):
```

Comment:
I'm not sure that this is needed in tobac, but I also don't have opposition to including it. It feels a bit too specific and not connected enough, but on the other hand it's well documented and solves a relatively common problem.